package.json

```diff
@@ -29,15 +29,15 @@
   ],
   "dependencies": {
     "cross-fetch": "^3.1.4",
-    "get-stdin": "^9.0.0",
-    "graphql": "^14.7.0",
+    "get-stdin": "^8.0.0",
+    "graphql": "^16.4.0",
```

Comment:
I wonder if this will be enough to handle visualization of newer features like interfaces implementing interfaces, or that will require follow-on work with the actual visualizer code.